lib/Runtime/Language/InterpreterLoop.inl

```diff
@@ -232,15 +243,27 @@ SWAP_BP_FOR_OPCODE:
                 break;
             }
             case INTERPRETER_OPCODE::ExtendedMediumLayoutPrefix:
+            case INTERPRETER_OPCODE::DblExtendedMediumLayoutPrefix:
             {
 #ifndef INTERPRETER_ASMJS  // Asmjs doesn't have any extended opcodes for now, remove that case
```

Comment:
@MikeHolman, I see Extended opcodes for Medium/Large layouts are disabled for AsmJs interpreter. Is that still valid ? If so, then perhaps those case arms should be removed entirely since neither Asmjs or normal interpreter use them ?